### PR TITLE
docs: fix pullapprove url

### PR DESCRIPTION
**Pull request by NejcZdovc**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
Fixes link in docs to file for `PullApprove`

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by NejcZdovc** [comment]
@googlebot I signed it!

File: docs/COMMITTER.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ As a contributor, see the instructions in [CONTRIBUTING.md](../CONTRIBUTING.md).
 
 # Change approvals
 
-Change approvals in our monorepo are managed via [PullApprove](https://docs.pullapprove.com/) and are configured via the [`.pullapprove`](../.pullapprove) file.
+Change approvals in our monorepo are managed via [PullApprove](https://docs.pullapprove.com/) and are configured via the [`.pullapprove`](../.pullapprove.yml) file.
```
Comment (gkalpak) [inline docs/COMMITTER.md:10]:
```suggestion
Change approvals in our monorepo are managed via [PullApprove](https://docs.pullapprove.com/) and are configured via the [`.pullapprove.yml`](../.pullapprove.yml) file.
```

**Review by gkalpak — CHANGES_REQUESTED** [review]
Good catch! Thx, @NejcZdovc :+1:
Can you please also update the file name in the link text?

**Comment by NejcZdovc** [comment]
@gkalpak  changed text as well

**Review by gkalpak — APPROVED** [review]
LGTM 🚀

**Pull request closed without merging**